backend/internal/storage/postgres/classrooms.go

```diff
@@ -174,10 +189,11 @@ func (db *DB) ModifyUserStatus(ctx context.Context, classroomID int64, status mo
 
 func (db *DB) GetUsersInClassroom(ctx context.Context, classroomID int64) ([]models.ClassroomUser, error) {
 	rows, err := db.connPool.Query(ctx, `
-	SELECT u.id, u.first_name, u.last_name, u.github_username, u.github_user_id, cm.classroom_id, cm.classroom_role as role, cm.status as status
+	SELECT u.id, u.first_name, u.last_name, u.github_username, u.github_user_id, cm.classroom_id, cm.classroom_role, cm.status, c.name as classroom_name, c.created_at as classroom_created_at, c.org_id, c.org_name
 	FROM users u
 	JOIN classroom_membership cm ON u.id = cm.user_id
-	WHERE cm.classroom_id = $1`, classroomID)
+	JOIN classrooms c ON c.id = cm.classroom_id
+	WHERE cm.classroom_id = $1 AND cm.status != 'REMOVED'`, classroomID)
```

Comment:
Should we define enum for ClassroomMembership statuses?

Reply:
exists. added in latest fix